networking/synthetic/stac-test.yaml

```diff
@@ -8,9 +8,21 @@
     
   - name: Copy Dockerfile to producer and consumer
     copy: src=stac/Dockerfile dest=/tmp/stac/Dockerfile
+
+  - name: Copy onload rpms to producer and consumer
+    copy: src=stac/openonload-201606_u1.2-1.el7.x86_64.rpm dest=/tmp/stac/openonload-201606_u1.2-1.el7.x86_64.rpm
+    
+  - name: Copy onload rpms to producer and consumer
+    copy: src=stac/openonload-kmod-3.10.0-514.el7-201606_u1.2-1.el7.x86_64.rpm dest=/tmp/stac/openonload-kmod-3.10.0-514.el7-201606_u1.2-1.el7.x86_64.rpm
+
+  - name: Create stac tuned profile directory if not exists
+    file: path=/usr/lib/tuned/stac state=directory
```

Comment:
/usr/lib/tuned is owned by the tuned RPM

custom tuned profiles must go in /etc/tuned/